packages/api/test/getNetworksConfig.spec.ts

```diff
@@ -0,0 +1,64 @@
+import fs from 'fs'
+import path from 'path'
+import { normalizeNetworkConfig } from '../src/utils/index'
+
+describe('validateDataFeedsConfig', () => {
+  it('check if the structure is correct', async () => {
+    const dataFeedsRouterConfig = JSON.parse(
+      fs.readFileSync(
+        path.resolve('./test/web3Middleware/dataFeedsRouter.json'),
+        'utf-8'
+      )
+    )
```

Comment:
I think you can import this without using `fs.readFileSync`
```suggestion
import dataFeedsRouterConfig from '.test//web3Middleware/dataFeedsRouter.json'
```